spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'hawk'
+require 'byebug'
```

Comment:
Not all organisations gallop as swiftly to the bleeding edge as IFAD, @lucaspiller :-P